apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/service/NamespaceServiceTest.java

```diff
@@ -98,6 +108,10 @@ public void testFindNamespace() {
                                           testNamespaceName)).thenReturn(someRelease);
```

Comment:
with the new logic, I think the following mock setup could be removed?

```java
    when(releaseService.loadLatestRelease(testAppId, Env.DEV, testClusterName,
                                          testNamespaceName)).thenReturn(someRelease);
    when(releaseService.loadLatestRelease(testAppId, Env.DEV, testClusterName, "hermes")).thenReturn(someRelease);
    when(itemService.findItems(testAppId, Env.DEV, testClusterName, testNamespaceName)).thenReturn(someItems);
```

Reply:
Removed